requirements.dev.txt

```diff
@@ -6,24 +6,20 @@
 #
 attrs==25.3.0
     # via hypothesis
-black==24.8.0
-    # via -r requirements.dev.in
 build==1.2.2.post1
     # via pip-tools
 certifi==2025.1.31
     # via
-    #   -c /home/runner/work/job-runner/job-runner/requirements.prod.txt
+    #   -c /home/tom/Documents/bennett-institute/job-runner/requirements.prod.txt
```

Comment:
nit: I tend to discard these changes that pip-tools (I think) makes. But they'll get overwritten the next time the workflow updates these dependencies anyway